docs/technical_reference/unit_models/uv_aop.rst

```diff
@@ -0,0 +1,80 @@
+Advanced oxidation processes (AOPs) are technologies involving the generation of highly reactive oxidative species, predominantly hydroxyl radicals (:math:`\text{HO} \cdot`). Unlike conventional chemical oxidation processes, such as using chlorine, which are selective as to which compounds they can degrade, AOPs are able to completely convert organic compounds into carbon dioxide, water and mineral acids. In addition, AOPs are feasible for full-scale use to destroy organic compounds because they generate hydroxyl radicals at ambient temperature and atmospheric pressure. Typically, the commercially available UV AOPs for industrial water treatment are:
+
+1) UV light and hydrogen peroxide (:math:`\text{H}_2\text{O}_2`)
+
+2) UV light and ozone (:math:`\text{O}_3`)
+
+3) UV light, hydrogen peroxide (:math:`\text{H}_2\text{O}_2`) and ozone (:math:`\text{O}_3`)
+
+UV AOPs can be  modeled at several different levels, depending on the known kinetic pathways and the modeling objectives. In this work, a basic level kinetic model is presented with an assumption on pseudo-steady state approximation for the kinetic description of free radical species. A pseudo-first order rate constant is utilized to represent the overall degradation rate of contaminants. The users need to provide either UV dose and disinfection rate, or rate constant and exposure time that are can be acquired during UV validation tests. These measurements are then used to simulate the removal of contaminants during the UV AOP process.
+
+This model also accounts for the energy demand for the disinfection process. The users need to provide Electrical Efficiency per Log Order Reduction (EE/O) to evaluate energy costs.
+
+Ports
+-----
+
+The model provides two ports (Pyomo notation in parentheses):
+
+* Inlet port (inlet)
+* Outlet port (outlet)
+
+Sets
+----
+.. csv-table::
+   :header: "Description", "Symbol", "Indices"
+
+   "Time", ":math:`t`", "[0]"
+   "Phases", ":math:`p`", "['Liq']"
+   "Components", ":math:`j`", "['H2O', 'NDMA', ...]"
+
+Degrees of Freedom and Variables
+--------------------------------
+Aside from the inlet feed state variables (i.e., temperature, pressure, component mass flowrates),
+the UV AOP model has at least an additional 5 degrees of freedom that
+the user must specify. The table below gives an outline of these.
+
+.. csv-table::
+   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
+
+   "Inlet temperature", ":math:`T`", "temperature", "[t]", ":math:`\text{K}`"
+   "Inlet pressure", ":math:`P_{in}`", "pressure", "[t]", ":math:`\text{Pa}`"
+   "Outlet pressure", ":math:`P_{out}`", "pressure", "[t]", ":math:`\text{Pa}`"
+   "Mass flowrate of components", ":math:`M_j`", "flow_mass_phase_comp", "[t, 'Liq', j]", ":math:`\text{kg/s}`"
+   "Inlet volumetric flowrate", ":math:`F_{in}`", "flow_vol", "[t]", ":math:`\text{m}^3\text{/s}`"
+   "Inactivation rate coefficient", ":math:`k`", "inactivation_rate", "['Liq', j]", ":math:`\text{m}^2\text{/J}`"
```

Comment:
When stating that the DOF is 5 after specifying the inlet state, is it correct only the variables starting at Inactivation rate coefficient and below would contribute to these 5 as the ones above it are all coupled to the inlet state? It may be worth separating these tables since the first 5 are based on state properties and the following 8 are the true DOF of the model, where 5 of 8 must be fixed. Assuming I am interpreting the model correctly.

Reply:
Do outlet pressure and flow_vol not also come from the property package when referring to the footnote?

Reply:
I agree- have a look at some of the older documentation for RO and how we mention the need to fix inlet feed conditions but focus explicitly on the unit model variables that must be fixed. Then you can revise this accordingly.

Reply:
You are correct that the variables after inactivation rate are the 5 additional DOF. I followed the `coag_floc_model.rst` which is also a single input single output model for the documentation. I think the note has already explained those first variables belong to the property package, similar to the `coag_floc_model.rst`.
As you suggested, I added `flow_vol` in the notes, and both inlet and outlet pressure has the same notation `pressure`.

Reply:
Al right, I separated the table.